smtk/extension/qt/qtUIManager.h

```diff
@@ -48,15 +48,18 @@ namespace smtk
     Q_OBJECT
 
     public:
-    qtUIManager(smtk::attribute::System &system,
-                const std::string &toplevelViewName);
+    qtUIManager(smtk::attribute::System &system);
       virtual ~qtUIManager();
 
       void initializeUI(QWidget* pWidget, bool useInternalFileBrowser=false);
-      qtBaseView* initializeView(QWidget* pWidget, smtk::common::ViewPtr view,
-        bool useInternalFileBrowser=true);
+      qtBaseView *setSMTKView(smtk::common::ViewPtr v);
```

Comment:
The name of these methods may be better to be like these?
setSMTKView() => setTopView()
smtkView() => topView()